apps/admin_panel/assets/src/omg-page-activity-log/index.js

```diff
@@ -243,6 +257,7 @@ class AccountPage extends Component {
           loadingStatus={individualLoadingStatus}
         >
           <SortableTable
+            activeIndexKey={this.state.selectedRow}
```

Comment:
const activeIndexKey = queryString.parse(this.props.location.search)['show-activity-tab']
then you can remove all the state